editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -1,15 +1,20 @@
 created: 20131205155227468
-modified: 20140919191220377
+custom-list: [[Headings in WikiText]] [[Paragraphs in WikiText]] [[Hard Linebreaks in WikiText]] [[Formatting in WikiText]] [[Dashes in WikiText]] [[Block Quotes in WikiText]] [[Lists in WikiText]] [[Definitions in WikiText]] [[Horizontal Rules in WikiText]] [[Tables in WikiText]] [[Code Blocks in WikiText]] [[Typed Blocks in WikiText]] [[Images in WikiText]] [[Linking in WikiText]] [[Variables in WikiText]] [[Macro Definitions in WikiText]] [[Macro Calls in WikiText]] [[Macros in WikiText]] [[Transclusion in WikiText]] [[Transclusion and Substitution]] [[Styles and Classes in WikiText]] [[HTML in WikiText]] [[Widgets in WikiText]] [[WikiText Parser Modes]]
+modified: 20220727095329372
 tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
 
-~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[MarkDown|http://daringfireball.net/projects/markdown/]], but with more of a focus on linking and the interactive features.
+~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of MarkDown but with more of a focus on linking and the interactive features.
 
 ~WikiText can also be inserted to the text field using the [[Editor toolbar]].
 
-See [[Formatting text in TiddlyWiki]] for an introduction to WikiText.
+The following list of syntax rules ''start simple and get more complex'' at the end of the list.
 
-The following elements of WikiText syntax are built into the core:
+<$button actions=<<wikitext-open-custom-list>> >Open all tiddlers from the list below</$button>
```

Comment:
My point here was that we should not single out this tiddler for a special "open all children" button. If it is useful then we should add it consistently across all tiddlers, and not just for this one. In any case it should be a separate PR

Reply:
We constantly get feedback from new users, that strict alphabetical listing doesn't make sense for them and the "non linearity" is overwhelming at the beginning, because they don't know, where to start and where to end. 

That's why I did try to create a "User Story" by rearranging the list to have the "Basic topics" at the beginning and the "Advanced" at the second part of the list. 

![image](https://user-images.githubusercontent.com/374655/181593313-04f64c3d-4936-429f-9857-13405d7d85fd.png)

If a new user wants to get an overview of all possible wikitext rules, they need the possibility to open the "User Story", in an order they can understand.

They can read the whole story from top to bottom and get info from simple to complex. There is no other way at the moment to accomplish this task in a convenient way. ... The "open all" button should be that way. ... May be it should be named "Open the WikiText user story"

>.. If it is useful then we should add it consistently across all tiddlers, and not just for this one. In any case it should be a separate PR

The `<<wikitext-open-custom-list>>` macro should be a consistent way to get "Story Lists". At the moment it uses the "custom-list" field to store that order. ... May be we should name it "user-story" 

This PR is part of the documentation branch. So it can be changed quickly. If it doesn't work out we can change it again. 

BUT if we don't change anything to get feedback from the community, we don't know, how we can improve the rest of the docs, so new users are comfortable with it. --- The only feedback we can get is: "It's way to complicated"

This PR should have been the "first step" to improve readability and approachability for new users. The [Overview](https://github.com/Jermolene/TiddlyWiki5/issues/6829) shows more topics, which I think could help us. It would add more interactivity, so users can try out the syntax while reading it, with 2 windows open side by side. 

If it is impossible to change the structure of the documentation. How should the community improve it? ... They could add one more thread at Talk, with a hypothetical discussion, how to improve the documentation.

May be we decide that Talk should be the medium to be used to document TiddlyWiki, using markdown instead of wikitext. .. But that's not what I want.